packages/stargate/src/stargateclient.ts

```diff
@@ -273,6 +276,31 @@ export class StargateClient {
     return this.forceGetQueryClient().bank.allBalances(address);
   }
 
+  public async getBalanceStaked(address: string): Promise<Coin | null> {
+    const allDelegations = [];
+    let startAtKey: Uint8Array | undefined = undefined;
+    do {
+      const { delegationResponses, pagination }: QueryDelegatorDelegationsResponse =
+        await this.forceGetQueryClient().staking.delegatorDelegations(address, startAtKey);
+
+      const loadedDelegations = delegationResponses || [];
+      allDelegations.push(...loadedDelegations);
+      startAtKey = pagination?.nextKey;
+    } while (startAtKey?.length !== 0 && startAtKey !== undefined);
```

Comment:
You can swap those to make it a bit easier to understand

```suggestion
    } while (startAtKey !== undefined && startAtKey.length !== 0);
```

This should compile because `startAtKey` is known to be defined after `&&`.